docs/4.3/kubernetes_ssh.md

```diff
@@ -34,49 +30,90 @@ proxy_service:
         public_addr: [teleport.example.com:3026]
         listen_addr: 0.0.0.0:3026
 ```
-
 Let's take a closer look at the available Kubernetes settings:
 
-* `public_addr` defines the publicly accessible address which Kubernetes API
-  clients like `kubectl` will connect to. This address will be placed inside of
-  `kubeconfig` on a client's machine when a client executes `tsh login` command
-  to retrieve its certificate. If you intend to run multiple Teleport proxies behind
-  a load balancer, this must be the load balancer's public address.
+- `public_addr` defines the publicly accessible address which Kubernetes API clients
+  like `kubectl` will connect to. This address will be placed inside of kubeconfig on
+  a client's machine when a client executes tsh login command to retrieve its certificate.
+  If you intend to run multiple Teleport proxies behind a load balancer, this must
+  be the load balancer's public address.
 
-* `listen_addr` defines which network interface and port the Teleport proxy server
+- `listen_addr` defines which network interface and port the Teleport proxy server
   should bind to. It defaults to port 3026 on all NICs.
 
-## Connecting the Teleport proxy to Kubernetes
+### Connecting the Teleport proxy to Kubernetes
+
+There are two options for setting up Teleport to access Kubernetes:
 
-There are two ways this can be done:
+## Deploy Inside Kubernetes as a pod
```

Comment:
Can we maybe add "Option 1:" to this heading to make it clear that there are two mutually exclusive options? If someone is skimming the docs it might make it jump out at them a bit more.